lib/sinon/util/fake_timers.js

```diff
@@ -2,17 +2,30 @@
 
 var llx = require("lolex");
 
-exports.useFakeTimers = function () {
-    var now;
-    var methods = Array.prototype.slice.call(arguments);
+/**
+ *
+ * @param config {number|Date|Object} the unix epoch value to install with (default 0) or
+ */
+exports.useFakeTimers = function (args) {
+    var config = {};
 
-    if (typeof methods[0] === "string") {
-        now = 0;
+    if (typeof args === "undefined" || args === null) {
+        config.now = 0;
+    } else if ((typeof args === "number" || args instanceof Date) && arguments.length === 1) {
+        config.now = args;
+    } else if (args !== null && typeof args === "object" && arguments.length === 1) {
+        var keys = Object.keys(args);
+        for (var i = 0; i < keys.length; i++) {
+            if (args.hasOwnProperty(keys[i])) {
+                config[keys[i]] = args[keys[i]];
+            }
+        }
     } else {
-        now = methods.shift();
+        throw new TypeError("useFakeTimers expected epoch or config object." +
```

Comment:
The error message doesn't need to be two lines, and "see" should start with capitalised "S".